cascade.py

```
@@ -1,146 +1,336 @@
+                logging.warning(f"Camera process exited with code {exitcode}. Restarting in 5 seconds.")
+                time.sleep(5)
+                # Optionally clear main_deque here if you want
+                sq_cascade.camera_process = multiprocessing.Process(
+                    target=camera_process_entry,
+                    args=(sq_cascade.main_deque, sq_cascade.camera_id, sq_cascade.shutdown_flag, sq_cascade.pause_event, sq_cascade.pause_duration, sq_cascade.log_level_str),
+                    daemon=True
+                )
+                sq_cascade.camera_process.start()
+                with suppress(Exception):
+                    bot_instance.send_text("🔄 Camera process restarted due to repeated RTSP failures.")
+                    if (camera_host, camera_ssh_username, camera_remote_command, camera_ssh_key_file):
```

Ruff (0.12.2)

If test is a tuple, which is always `True`

(F634)